src/tabs/tab-header-bar.tsx

```diff
@@ -487,7 +487,7 @@ const TabTrigger = forwardRef(
     const children = (
       <>
         <span className={styles['tabs-tab-label']} ref={tabLabelRefObject}>
-          {tab.label}
+          <span>{tab.label}</span>
```

Comment:
A code comment here would be helpful as that span looks unnecessary from the code unless you know the detail.

Reply:
Oh definitely, I'll add one.